src/main/java/org/springframework/integration/aws/inbound/SqsMessageDrivenChannelAdapter.java

```diff
@@ -88,6 +88,10 @@ public void setResourceIdResolver(ResourceIdResolver resourceIdResolver) {
 		this.simpleMessageListenerContainerFactory.setResourceIdResolver(resourceIdResolver);
 	}
 
+	public void setAutoStartUp(boolean autoStartUp) {
+		this.simpleMessageListenerContainerFactory.setAutoStartup(autoStartUp);
+	}
```

Comment:
@artembilan I believe I've address these comments now, the unit tests are now passing.